lib/IRGen/IRGenSIL.cpp

```diff
@@ -2442,7 +2442,7 @@ void IRGenSILFunction::visitGraphOperationInst(GraphOperationInst *i) {
 
   // Set the device.
   opDevice = deviceInfo->handleDevicePlacement(opInfo.getOperationName(),
```

Comment:
from our discussion today, by the time this function is called, `i` should contain the pseudo attr `__device`, set in TFDeabstraction earlier via `deviceInfo->handleDevicePlacement()`, so we need not use the latter here.

This is an issue in pre-existing code, so pls feel free to leave a TODO